nix/default.nix

```diff
@@ -0,0 +1,43 @@
+{ system ? builtins.currentSystem }:
+let
+  pkgs = (import ./nixpkgs.nix {
+    overlays = [ (final: prev: {
+      pcre = prev.pcre.overrideAttrs (x: {
+        configureFlags = x.configureFlags ++ [ "--enable-static" ];
+      });
```

Comment:
To clarify if someone is wondering why we use an `overlay` vs a `config` (down below) here: `pcre` is part of `stdenv`, which does not allow us to change it without re-building all dependencies. The major reason why we need to do that is the presence of `glib2` in conmon, which relies on `pcre`.

I vote for removing `glib2` form conmon since it seems to bloat-up the whole binary. I'm happy to contribute that change if nobody has strong opinions against it.